src/hooks/useVotingPowerOnProposal.ts

```diff
@@ -0,0 +1,60 @@
+import useAsyncMemo from 'decentraland-gatsby/dist/hooks/useAsyncMemo'
+
+import { ProposalAttributes } from '../entities/Proposal/types'
+import { Vote } from '../entities/Votes/types'
+import { Scores, getProposalScores } from '../entities/Votes/utils'
+
+export const MINIMUM_VP_REQUIRED_TO_VOTE = 1
+
+function getDelegatedVotingPowerOnProposal(
+  scoresAtProposalCreation: Scores,
+  delegators: string[] | null,
+  votes: Record<string, Vote> | null | undefined
```

Comment:
This could be rewritten as `votes?: Record<string, Vote> | null` 

It is the same but I think it looks cleaner 😃